KIPs/kip-34.md

```diff
@@ -69,7 +69,7 @@ With the common architecture, we want to achieve two goals:
 
 This is the overview of the common architecture of Klaytn SDK.
 
-![whole](https://user-images.githubusercontent.com/32922423/106080346-35a6b200-615a-11eb-96be-44fffd382c81.png)
+![whole_20210217](https://user-images.githubusercontent.com/32922423/108157902-2a2a2380-7127-11eb-814f-b166f2983e9f.png)
```

Comment:
For both images (this and the one below), in the KIP-17 box, next to "MintwithTokens" there is a little typo "toeken"

Reply:
I fixed mistypo :)